docker_test.go

```diff
@@ -1506,6 +1506,18 @@ func TestContainerWithReaperNetwork(t *testing.T) {
 	assert.NotNil(t, cnt.NetworkSettings.Networks[networks[1]])
 }
 
+func TestGetGatewayIP(t *testing.T) {
```

Comment:
Thanks for contributing a unit test! This is super!

I'd like to ask about the assertions part: according to the PR description:

>the IP address in the Host Port wait strategy is incorrect

Is it enough to verify that no errors are raised, or do we need to verify any other thing? 

BTW It's not a blocker on the review, I just want to make sure we are covering the new behaviour with the tests.

Reply:
> Is it enough to verify that no errors are raised, or do we need to verify any other thing?

We are not sure about the specific value of the IP address in the GHA test or host test, so we can only verify that the IP address is not empty and there is no error. Do you think we also need to create a container with an export port to verify that this address is available?

Reply:
Yeah, if reproducing GHA behaviour is complex I'd keep it like it is now adding the verification of the IP address is not empty. If you can give it a try with a real container, best. But as I said, let's not block this unless needed.

Reply:
Have added the not empty check :)

Reply:
offtopic: it seems the `@dalekliuhan` commiter user does not have associated an email on github

Reply:
If the CI passes, this LGTM :)

Reply:
> offtopic: it seems the `@dalekliuhan` commiter user does not have associated an email on github

email address on GitHub? I have set it. Do I miss anything? :)

Reply:
Look at the commit in the list: it does not have an image associated to it. And when you click on the commit itself, the user is not clickable

Reply:
Oh, sorry about that, I submit by a different user, not the GitHub user so it cannot be clickable.

Reply:
@mdelapenya Does this repo support `squash merge`? If so, GitHub will take care of this issue automatically.